examples/stable_diffusion_v2/inference/export.py

```diff
@@ -0,0 +1,250 @@
+import argparse
+import logging
+import os
+import sys
+
+from omegaconf import OmegaConf
+
+import mindspore as ms
+from mindspore import ops
+
+workspace = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(workspace))
+from ldm.modules.logger import set_logger
+from ldm.util import instantiate_from_config, str2bool
+from libs.helper import load_model_from_config, set_env
+from libs.infer_engine.export_modules import (
+    Img2ImgDataPrepare,
+    InpaintDataPrepare,
+    InpaintPredictNoise,
+    NoisySample,
+    PredictNoise,
+    SchedulerPreProcess,
+    Text2ImgDataPrepare,
+    VAEDecoder,
+)
+
+logger = logging.getLogger("text to image speed up")
+
+
+def model_export(net, inputs, name, model_save_path):
+    ms.export(net, *inputs, file_name=os.path.join(model_save_path, name), file_format="MINDIR")
+    logger.info(f"convert {name} mindir done")
+
+
+def lite_convert(name, model_save_path, converter):
+    import mindspore_lite as mslite
+
+    mindir_path = os.path.join(model_save_path, f"{name}.mindir")
+    if not os.path.exists(mindir_path):
+        mindir_path = os.path.join(model_save_path, f"{name}_graph.mindir")
+    converter.convert(
+        fmk_type=mslite.FmkType.MINDIR,
+        model_file=mindir_path,
+        output_file=os.path.join(model_save_path, f"{name}_lite"),
+        config_file="./libs/infer_engine/sd_lite.cfg",
+    )
+    logger.info(f"convert {name} lite mindir done")
+
+
+def main(args):
+    # set logger
+    set_env(args)
+    ms.set_context(device_target="CPU")
```

Comment:
the parser's help says that "Device target, should be in [Ascend]", but here is set to CPU.

Reply:
The MindSpore export will occupy memory，resulting in lite conversion failure. The MindSpore MindIR is common to different devices, so set MindSpore run on CPU, Lite converter run on Ascend